pyresample/test/test_area_config.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
```

Comment:
Do these lines actually do anything useful for a test module?  As far as I know, the first line only matters for commandline scripts and the second is redundant in Python 3?

Reply:
I haven't thought about that tbh. For consistency, maybe we should have a PR just for removing all of these?

Reply:
Or I could do it here...

Reply:
We could probably remove them. I've only ever added them for consistency. As a library, we could probably remove the usage of both in all pytroll packages.